var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -27,6 +27,7 @@ class Seacas(CMakePackage):
 
     # ###################### Versions ##########################
     version('master', branch='master')
+    version('2021-09-29a', sha256='28becc97b1ad81c315e44ccdd0a154d054473ad05c3f28e4904736c4122771cc')
```

Comment:
Is this an alpha release? If so can you make the previous version preferred?

Reply:
No, the releases are tagged with the y-m-d and I made a bug fix after already tagging it as 2021-09-29, so added the 
`a` to distinguish.  I realize now that this looks like alpha....  I will create a new tag of `2021-09-30`  just so it follows the form of the previous tags.